src/main/java/net/mcreator/ui/views/editor/image/ImageMakerView.java

```diff
@@ -74,11 +77,15 @@ public class ImageMakerView extends ViewBase implements MouseListener, MouseMoti
 
 	private final VersionManager versionManager;
 
+	private final ClipboardManager clipboardManager;
+
 	private final JLabel imageInfo = new JLabel("");
 
 	public final JButton save;
 
 	public static final ExecutorService toolExecutor = Executors.newSingleThreadExecutor();

```

Comment:
If possible, move private static final entries to the top and others lower, and add new-lines to different "groups", eg. all that relates to panels, saving, cursors, ...